### PR TITLE
Text Fixes

### PR DESCRIPTION
**Pull request by SgtBears**
Cities, Civics and the start of the Civilizations.
I just deleted the cities file since none of the tags inside were ever used.
This includes translations, strategies and pedias.

Which one of these mercantilism help texts would you like to display ingame.
------ Increased chance of spreading Trading Company
------ Reduced expansion penalty for overseas colonies

### CODE REVIEW

File: Assets/XML/Text/Civics.xml [hunk 1]
```diff
@@ -180,7 +196,7 @@
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_CIVIC_LEVY_ARMIES_STRATEGY</Tag>
-		<English>Extra free units, military units produced with food, faster construction of Castles</English>
+		<English>Lower unit support costs; military units produced with food; double production speed for castles</English>
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_CIVIC_MERCANTILISM_HELP</Tag>
```
Comment (SgtBears) [inline Assets/XML/Text/Civics.xml]: Which Mercantilism Help would you like to be displayed. I will remove the other one.

**Comment by dguenms** [comment]
Neither, I think.

**Comment by SgtBears** [comment]
Okay I removed the mercantilism help from the civic file.

**Comment by dguenms** [comment]
Okay good, merged.

**Pull request merged**